.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: check HTML
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: Run a one-line script
+        run: echo Hello, world!
```

Comment:
I'm curious, is this needed for anything?

Reply:
hah, I forgot to remove it.